plugins/database/mssql/mssql_test.go

```diff
@@ -79,6 +81,38 @@ func TestInitialize(t *testing.T) {
 	}
 }
 
+func TestInitializeCloudSQL(t *testing.T) {
+	connURL := os.Getenv("CONNECTION_URL")
```

Comment:
How does this work for manual runs and CI?

Reply:
should we be skipping this for CI then?

Reply:
Yes, these would be skipped in CI since they explicitly depend on external resources being set up. We can make the updates to skip them in CI unless the env vars are set.

I think there's a lot of precedents for skipping tests like this in GCP, so hoping that this fits in with the rest.

Reply:
Removed this test since it was unnecessary in the case of MSSQL. Rest of the integration tests are being skipped in CI